springboottrades-web/src/main/resources/templates/fragments/header_fragment.html

```diff
@@ -1,12 +1,13 @@
 <html lang="en" xmlns="http://www.w3.org/1999/xhtml" xmlns:th="http://www.thymeleaf.org">
 <th:block th:fragment="header-includes">
 	<link rel="stylesheet" type="text/css" media="all"
-		href="../webjars/bootstrap/2.3.2/js/bootstrap.css"
-		th:href="@{webjars/bootstrap/2.3.2/css/bootstrap.css}" />
+		href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css" />
+	<link rel="stylesheet" type="text/css" media="all"
+		href="https://maxcdn.bootstrapcdn.com/font-awesome/4.4.0/css/font-awesome.min.css" />
 	<link rel="stylesheet" type="text/css" media="all" href="../static/css/style.css"
 		th:href="@{/css/style.css}" />
-	<script th:src="@{webjars/jquery/2.1.1/jquery.js}"></script>
-	<script th:src="@{webjars/bootstrap/2.3.2/js/bootstrap.js}"></script>
+	<script src="https://code.jquery.com/jquery-2.1.4.min.js"></script>
```

Comment:
Wondering if there is a way to hide the version numbers here so that whenever we upgrade boot that brings in new version we are not caught out by not changing version numbers in the path here.